tests/e2e_tests/test_topology.py

```
@@ -0,0 +1,139 @@
+import os
+import sys
+from playwright.sync_api import expect
+
+# Importing utilities for test assertions and setup
+from tests.e2e_tests.utils import (
+    trigger_alert,
```

Ruff (0.8.2)

`tests.e2e_tests.utils.trigger_alert` imported but unused

Remove unused import

(F401)